.claude/commands/polish.md

```
@@ -0,0 +1,185 @@
+| File/path | Focus on specific file(s) |
+
+Use the GitHub CLI to fetch PR details:
+
+```bash
+gh pr view [PR_NUMBER] --json title,body,state,mergeable,headRefName,baseRefName
+gh pr diff [PR_NUMBER]
+```
+
+If the `gh` CLI is not installed or authenticated, show:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)